client/src/CommentBox.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import $ from 'jquery';
+import CommentForm from './AddCommentForm';
```

Comment:
I don't understand ES6 export/import well enough to get how this works, but I think the names should match to be consistent.